lib/components/Application.js

```diff
@@ -83,44 +81,39 @@ export default class Application extends Component {
     const budgets = this.state.budgets
     firebase.database().ref(`users/${this.state.user.uid}/budgets/`).push({ itemValue, itemTime, budgetItem })
     firebase.database().ref(`users/${this.state.user.uid}/monthlyIncome`).push({ monthlyIncome })
-    this.setState({ budgets : { draftBudgetItem: '', draftItemValue: '', itemTime: moment(Date.now()).format('MM/D, h:mm a') } })
+    this.setState({ budgets: { draftBudgetItem: '', draftItemValue: '', itemTime: moment(Date.now()).format('MM/D, h:mm a') } })
   }
 
   render() {
     return (
-      <BrowserRouter>
+      <HashRouter basename="/public">
```

Comment:
yes @StephanieEA but in its own documentation its referred to as "hacky" lol so idk